buildpack-deps/Dockerfile.fedora.tpl

```diff
@@ -0,0 +1,26 @@
+FROM #{FROM}
+
+RUN dnf install -y \
+		autoconf \
+		make automake gcc gcc-c++ kernel-devel \
```

Comment:
there is no build-essential package on Fedora and this combination is the closest equivalent so I want to have them in one line